db/memtable_list.cc

```diff
@@ -726,6 +727,17 @@ Status InstallMemtableAtomicFlushResults(
     assert(0 == num_entries);
   }
 
+  if (vset->db_options()->allow_2pc) {
```

Comment:
Should the non-2pc case be handled too?

Reply:
The algorithm for computing the min_log_number has problems, the non-2pc case is fixed in https://github.com/facebook/rocksdb/pull/7660. I'll update this PR after https://github.com/facebook/rocksdb/pull/7660 is merged.